### ISSUE TITLE
Allow certain workers to stay in their workplace

### ISSUE THREAD

**Issue by pantsguyapp** [ldtteam/minecolonies-features#625]
### Prerequisites 

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [X] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my feature does not already exist.
- [X] I made sure that **this feature request is not a duplicate of any existing feature request.**

### Context 

- Minecraft Version:
- MineColonies Version:

## Is your feature request related to a problem?

there are some beds in certain schematic, shame that they are completely useless.

## Feature description

It makes sense for some worker to stay to their work, such as the **crafters**, as I see they stay in their house all day if there are nothing to do.

### Notes or related things

---
## Viewers

* Add a thumbs-up to the feature request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Talyda** [comment]
...you were already told that them wandering around was intended behaviour.

**Comment by pantsguyapp** [comment]
this thread doesn't mention anything about that thought, the crafters can still wander away wherever they feel like, it just that they now sleep in their hut instead.

**Comment by Talyda** [comment]
You also didn't mention having them sleep in their hut =P You said to have them stay in their work place.
Beds are in workplaces for decoration. They sleep in their houses. If you want them to sleep in their workplaces, you could make a schematic that combines a house and a workplace =)

**Comment by pantsguyapp** [comment]
Fair point, I edited it so that no one should make the same mistake as you did hopefully.
The bed is cosmetic but it would be nicer to see some variety going in the colony, and it is logical for crafters to make an entire floor/room of the building into their house and sleep in there.
Still, the crafters is purely my suggestion, there are beds in other huts as well, no one force you to do the crafters only.

**Comment by Talyda** [comment]
I didn't make a mistake =P  I responded to what you wrote.

That aside, if you want workers to sleep in their work places, then you can create a schematic that combines a house and a workplace - but bear in mind if you do that you will either have to leave it at level 1, so that only the worker sleeps in teh workplace, or resign yourself to 4 random people sleeping in there too =)

Could I also suggest joining discord, and searching in the suggestions channel there? You'll find that this has been suggested many times before, and that it is not how they want it to work at the moment =) It would save cluttering up the github anyway!

**Comment by pantsguyapp** [comment]
True, it was MY writting that caused the misinterpretation.
Yeah thats my point, I want the worker to stay in their workplace while still getting all the benefits of living in a L5 house, so I don't fond of making a custom L1 schematic.
The discord: I was in fact from the discord, but I purposely deleted all of my discord accounts because I don't want to have anything to do with discord in general.

**Comment by Thodor12** [comment]
Attached schematics label as this would require schematic reworks based on which buildings are affected as beds would have to become a requirement in certain schematics, which not every schematic has.

**Comment by Moobien** [comment]
Wont happen. Devs have chosen for this to not be a thing. Thodor you can close this.
